docs/user-guide/spin.rst

```diff
@@ -7,7 +7,10 @@ Original paper: https://arxiv.org/abs/2401.01335
 
 The NeMo framework supports efficient model alignment via the NeMo Aligner codebase.
 
-All algorithms in NeMo Aligner will work with any GPT based model that is from mcore(i.e in the config it has ``mcore_gpt=True``). For the purposes of this tutorial, we will go through the entire SPIN pipeline using the newly released `2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>`__.  This same tutorial also works for GPT models(such as LLaMa2) of any size.
+All algorithms in NeMo Aligner will work with any GPT based model that is from mcore(i.e in the config it has ``mcore_gpt=True``). For the purposes of this tutorial, we will go through the entire SPIN pipeline using the newly released `2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>`__.  This same tutorial also works for GPT models(such as LLaMa3) of any size.
+
```

Comment:
Move Line 6/6 to Line 11 (or abouts) and revise as a sentence. 

The original paper on Self-Play Fine-Tuning is here: 
`https://arxiv.org/abs/2401.01335 <https://arxiv.org/abs/2401.01335>`_.

Reply:
Reworded the sentence a bit. Please let me know if you have any comments

Reply:
LGTM